install_config/aggregate_logging.adoc

```diff
@@ -1022,6 +1022,11 @@ $ oc patch dc/logging-es-<suffix> \
 . Once these steps are taken, a local host mount can be applied to each replica as in this example
 (where we assume storage is mounted at the same path on each node) (change to `--selector component=es-ops` for Ops cluster):
 +
+=======
+Once you have completed these steps, you can apply a local host mount to each
+replica. The following example assumes storage is mounted at the same path
+on each node.
+
```

Comment:
@jhoyt-rh - missing ending =======.

Same issue as https://github.com/openshift/openshift-docs/pull/14429/files

Reply:
those ======= were an artifact of the cherrypicking that I didn't notice. It has now been corrected.